docs/metrics/customizing-the-sdk/README.md

```diff
@@ -335,6 +335,61 @@ by using Views.
 
 See [Program.cs](./Program.cs) for a complete example.
 
+### Explicit Bucket Histogram Aggregation with .NET 9 Advice API
```

Comment:
thus document is about sdk customization, so histogram advice is somewhat misfit here. we can mention it in the views section about custom buckets, but anything more should be elsewhere.

Reply:
this is unresolved.